src/test/java/com/codurance/retropolis/acceptance/board/BoardStepDefsIntegrationTest.java

```diff
@@ -32,7 +33,7 @@ public void cleanUpDatabase() throws SQLException {
 
   @When("^the client calls /boards/(\\d+)")
   public void theClientCallsBoard(int boardId) {
-    HttpWrapper.executeGet("http://localhost:5000/boards/" + boardId);
+    HttpWrapper.executeGet(url + "/boards/" + boardId);
```

Comment:
This now means that it is easier if you are running with a different port locally. Great!